protocol-configuration/permissions.ts

```diff
@@ -31,5 +30,9 @@ export const permissions = {
   PSM_ADMIN_ROLE: [],
   TRIBAL_CHIEF_ADMIN_ROLE: ['optimisticTimelock', 'tribalChiefSyncV2'],
   VOTIUM_ADMIN_ROLE: ['opsOptimisticTimelock'],
-  PCV_GUARDIAN_ADMIN_ROLE: ['optimisticTimelock']
+  PCV_GUARDIAN_ADMIN_ROLE: ['optimisticTimelock'],
+  METAGOVERNANCE_VOTE_ADMIN: ['feiDAOTimelock', 'opsOptimisticTimelock'],
+  METAGOVERNANCE_TOKEN_STAKING: ['feiDAOTimelock', 'opsOptimisticTimelock'],
+  METAGOVERNANCE_GAUGE_ADMIN: ['feiDAOTimelock', 'opsOptimisticTimelock'],
+  METAGOVERNANCE_GAUGE_STAKING: ['feiDAOTimelock', 'opsOptimisticTimelock']
```

Comment:
Are any of these higher clearance?

Reply:
arguably `METAGOVERNANCE_GAUGE_ADMIN` is higher risk because it can stake the protocol's LP tokens held on the contract on an arbitrary contract (it can whitelist a gauge & then stake in it), so I'll move it to the optimisticTimelock that has a longer duration